tests/integration/tier1/destructive/one-kernel-scenario/test_one_kernel_scenario.py

```diff
@@ -86,5 +86,3 @@ def test_one_kernel_scenario(shell, convert2rhel, one_kernel):
         shell("yum-config-manager {}".format(enable_repo_opt))
 
         assert shell("yum install -y python3 --enablerepo=*").returncode == 0
-
```

Comment:
This `test_one_kernel_scenario` should also be in try except no? The `one_kernel` fixture also uses tmt-reboot

Reply:
Good point, but the reboot is running before the conversion, so if anything goes sideways during the conversion, tmt will be able to pick up the returncode from the pytest/convert2rhel run.
On the other hand, it probably won't pick up anything during the one_kernel() fixture :shrug:

Reply:
I see, as we talked in person - please remove the asserts from the `one_kernel` as they are pointless there anyway

Reply:
I've added the try except to the fixture, so we bail out when the setup fails